components/layout-wrapper.tsx

```diff
@@ -41,30 +41,54 @@ const LayoutWrapper = ({ children }: Props) => {
             </Link>
             <ul className="relative hidden pt-8 md:flex">
               {leftHeaderNavLinks.map((link) => (
-                <li key={link.title}>
+                <>
+                  <li key={link.title}>
+                    <Link
+                      key={link.title}
+                      href={link.href}
+                      className="px-1.5 lg:px-2 xl:px-3 font-medium transition-colors duration-200 hover:text-gray-600"
+                    >
+                      <span>{link.title}</span>
```

Comment:
Currently the left and right menu items are right up against each other. I'm not sure if we want any space between the two sections to distinguish them?

<img width="764" alt="Screenshot 2023-08-16 at 09 36 11@2x" src="https://github.com/tailscale-dev/tailscale-dev/assets/303731/f7f80793-22b1-4f09-a475-d80ccfea6262">

Reply:
I think there's a bigger issue of too much stuff in the header in general. I did some number twiddling and this is the best compromise I can get at the moment. I filed #292 to track fixing this. we may need to redesign the navbar in general.